apps/nextra/pages/en/build/indexer/common-queries/token-metadata.mdx

```diff
@@ -0,0 +1,30 @@
+import { GraphQLEditor } from 'components/index'
+import { Callout } from 'nextra/components'
+
+# Get Token Metadata by Name
+
+`GetTokensDataByName` retrieves metadata URIs for tokens by their names within a specified collection. This query is particularly useful for marketplaces to show metadata from tokens within a collection.
+
+<Callout type="info">
+Explore the capabilities of this query! Modify the variables below to tailor the query to your needs.
+</Callout>
+
+<GraphQLEditor
+  query={`query GetTokensDataByName($name: String, $collectionName: String) {
+  current_token_datas(
+    where: {name: {_eq: $name}, collection_name: {_eq: $collectionName}}
+  ) {
+    metadata_uri
+    __typename
+  }
+}`}
+  variables={`{
+  "name": "EpicDragon",
+  "collectionName": "MythicDragons"
```

Comment:
This query isn't returning any data given these variables. Can we swap them to be something else? Maybe

```suggestion
  "name": "The Mexican",
  "collectionName": "Aptomingos"
```